pyartcd/pyartcd/constants.py

```diff
@@ -48,3 +48,5 @@
 }
 
 GITHUB_OWNER = "openshift-eng"
+
+OTEL_EXPORTER_OTLP_ENDPOINT = "http://otel-collector-psi-rhv.hosts.prod.psi.rdu2.redhat.com:4317"
```

Comment:
Duplicated in doozerlib.constants.py. If we change endpoint, there are 2 places in the code to fix. The solution is moving this constant in artcommonlib.constants.py